src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,28 @@
 import React from 'react';
+// import commentsFromServer from '../../api/comments';
```

Comment:
There is a commented import statement. If it's not needed, please remove it to keep the code clean 🧹.